manifests/config.pp

```diff
@@ -70,5 +97,25 @@
       notify  => Class['confluence::service'],
     }
 
+    file { "${confluence::webappdir}/confluence/WEB-INF/web.xml":
+      source  => 'puppet:///modules/confluence/web.xml',
+      mode    => '0600',
+      owner   => $::confluence::user,
+      group   => $::confluence::group,
+      require => Class['confluence::install'],
+      notify  => Class['confluence::service'],
+    }
+
+  }
+  # if JDBC was configured along with the license key and server_id, skip some server setup steps.
+  if $confluence::tomcat_jdbc_settings and $confluence::license and $confluence::server_id {
+    file { "${confluence::homedir}/confluence.cfg.xml":
```

Comment:
is tomcat_jdbc_settings undef by default, or empty? empty strings are true. Also, why are these using variables from the main class instead of the class parameters (which get their default values from the main class anyway)?